drivers/adults_with_children_twentyfive_plus_hoh_sub_pop/extensions/grda_warehouse/hud/client_extension.rb

```diff
@@ -0,0 +1,21 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module AdultsWithChildrenTwentyfivePlusHohSubPop::GrdaWarehouse::Hud
+  module ClientExtension
+    extend ActiveSupport::Concern
+    include ArelHelper
+
+    included do
+      scope :adults_with_children_twentyfive_plus_hoh, -> do
+        where(
+          GrdaWarehouse::ServiceHistoryEnrollment.entry.adults_with_children_twentyfive_plus_hoh.
```

Comment:
This isn't right because it's looking only at age at entry....

Reply:
Leaving as is for now. It is consistent with the other sub-population calculations.